File: GraphicOverlayDemo/app/src/main/java/edu/cs4730/graphicoverlaydemo/SimpleFragment.java
package edu.cs4730.graphicoverlaydemo;


import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import edu.cs4730.graphicoverlaydemo.databinding.FragmentSimpleBinding;

/**
 * A simple {@link Fragment} subclass.
 */
public class SimpleFragment extends Fragment {
    FragmentSimpleBinding binding;

    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        binding = FragmentSimpleBinding.inflate(inflater, container, false);
        // Inflate the layout for this fragment
        return binding.getRoot();
    }

}

File: GraphicOverlayDemo_kt/app/src/main/java/edu/cs4730/graphicoverlaydemo_kt/SimpleFragment.kt
package edu.cs4730.graphicoverlaydemo_kt

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import edu.cs4730.graphicoverlaydemo_kt.databinding.FragmentSimpleBinding

/**
 * A simple [Fragment] subclass.
 */
class SimpleFragment : Fragment() {
    private lateinit var binding: FragmentSimpleBinding
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        // Inflate the layout for this fragment
        binding = FragmentSimpleBinding.inflate(inflater, container, false)
        return binding.root
    }
}